services/121-service/src/utils/token/token-validation.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@nestjs/common';
+import { TokenSet } from 'openid-client';
+
+@Injectable()
+export class TokenValidationService {
+  public isTokenValid(
+    tokenSet: TokenSet,
+    expiresAtInSeconds = false, // If true, then expires_at is assumed in milliseconds, as it should be
```

Comment:
This is very confusing: so when expiresAtInSeconds = true then it is in milliseconds??
Why "as it should be"?

I suggest that we put a bit more effort into this and solve it to be clean and simple. Ideas:
1. Create our own TokenSet sub-class with at least a comment that expires_at is "in milliseconds". But what does that even mean? A timestamp according to standard blabla? And if we can, add a validation that if someone passes in seconds instead of milliseconds, it starts to complain. Like: this timestamp is likely false, because the date is in the 18th century.
2. Get rid of this boolean parameter in this function.
3. Refactor IntersolveVisa code so that it also works in milliseconds.

Reply:
I opted for option 3, which seems quite feasible in the end. That way this parameter can be dropped completely.